### PR TITLE
implemented some hackfest feedback

### PR DESCRIPTION
**Pull request by klgill**
I implemented most of the feedback from the Hackfest Google doc.
https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit

### CODE REVIEW

File: docs_user/modules/proc_adopting-image-service-with-nfs-backend.adoc [hunk 1]
```diff
@@ -119,7 +119,7 @@ EOF
 . Patch the `OpenStackControlPlane` CR to deploy the {image_service} with an NFS back end:
 +
 ----
-$ oc patch openstackcontrolplane openstack --type=merge --patch-file glance_nfs_patch.yaml
+$ oc patch openstackcontrolplane openstack-galera-network-isolation --type=merge --patch-file glance_nfs_patch.yaml
```
Comment (klgill) [inline docs_user/modules/proc_adopting-image-service-with-nfs-backend.adoc:122]: This was called out in a few procedures. But `openstack` should not be changed to `openstack-galera-network-isolation` in every case, right?

Link to hackfest comment: https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABQ54c_1g

File: docs_user/modules/proc_adopting-key-manager-service.adoc [hunk 2]
```diff
@@ -38,7 +38,6 @@ spec:
       serviceAccount: barbican
       serviceUser: barbican
       passwordSelectors:
-        database: BarbicanDatabasePassword
```
Comment (klgill) [inline docs_user/modules/proc_adopting-key-manager-service.adoc:41]: Hackfest comment for reference: https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABQ54dA3g

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc [hunk 3]
```diff
@@ -78,10 +78,13 @@ spec:
...
               spec:
                 type: LoadBalancer
-        networkAttachments:
+        networkAttachments: <2>
         - storage
 '
 ----
++
+<1> Must match the {rhos_acro} deployment storage class.
+<2> Must match the network attachment for the previous {object_storage} configuration from the {OpenStackShort} deployment.
```
Comment (klgill) [inline docs_user/modules/proc_adopting-the-object-storage-service.adoc:88]: Hackfest comments for reference: 

- https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABRtZ2dcA
- https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABRtZ2db8

File: docs_user/modules/proc_deploying-backend-services.adoc [hunk 4]
```diff
@@ -112,6 +112,7 @@ $ oc set data secret/osp-secret "HeatPassword=$HEAT_PASSWORD"
 $ oc set data secret/osp-secret "IronicPassword=$IRONIC_PASSWORD"
 $ oc set data secret/osp-secret "IronicInspectorPassword=$IRONIC_PASSWORD"
 $ oc set data secret/osp-secret "ManilaPassword=$MANILA_PASSWORD"
+$ oc set data secret/osp-secret "MetadataSecret=$METADATA_SECRET"
 $ oc set data secret/osp-secret "NeutronPassword=$NEUTRON_PASSWORD"
 $ oc set data secret/osp-secret "NovaPassword=$NOVA_PASSWORD"
```
Comment (klgill) [inline docs_user/modules/proc_deploying-backend-services.adoc:117]: Hackfest comment for reference: https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABRuPpC2k

File: docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc [hunk 5]
```diff
@@ -84,26 +84,22 @@ After the {OpenStackShort} control plane services are shut down, if any of the e
...
-    "select host from nova.services where services.binary='nova-compute';")"
-PULL_OPENSTACK_CONFIGURATION_NOVAMANAGE_CELL_MAPPINGS="$($CONTROLLER1_SSH sudo podman exec -it nova_api nova-manage cell_v2 list_cells)"
+$ cat >~/.source_cloud_exported_variables <<EOF
+PULL_OPENSTACK_CONFIGURATION_DATABASES="$PULL_OPENSTACK_CONFIGURATION_DATABASES"
+PULL_OPENSTACK_CONFIGURATION_MYSQLCHECK_NOK="$PULL_OPENSTACK_CONFIGURATION_MYSQLCHECK_NOK"
+PULL_OPENSTACK_CONFIGURATION_NOVADB_MAPPED_CELLS="$PULL_OPENSTACK_CONFIGURATION_NOVADB_MAPPED_CELLS"
+PULL_OPENSTACK_CONFIGURATION_NOVA_COMPUTE_HOSTNAMES="$PULL_OPENSTACK_CONFIGURATION_NOVA_COMPUTE_HOSTNAMES"
+PULL_OPENSTACK_CONFIGURATION_NOVAMANAGE_CELL_MAPPINGS="$PULL_OPENSTACK_CONFIGURATION_NOVAMANAGE_CELL_MAPPINGS"
+SRIOV_AGENTS="$SRIOV_AGENTS"
 EOF
```
Comment (klgill) [inline docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc:94]: Hackfest comment for reference: https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABQ54dAj8

File: docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc [hunk 6]
```diff
@@ -84,26 +84,22 @@ After the {OpenStackShort} control plane services are shut down, if any of the e
...
 ----
-$ podman run -i --rm --userns=keep-id -u $UID $MARIADB_IMAGE mysql \
-    -rsh "$SOURCE_MARIADB_IP" -uroot -p"$SOURCE_DB_ROOT_PASSWORD" ovs_neutron -e \
-    "select host, configurations from agents where agents.binary='neutron-sriov-nic-agent';"
+SRIOV_AGENTS=$ oc run mariadb-client mysql -rsh "$SOURCE_MARIADB_IP" \
+-uroot -p"$SOURCE_DB_ROOT_PASSWORD" ovs_neutron -e \
+"select host, configurations from agents where agents.binary='neutron-sriov-nic-agent';"
 ----
+
+. Save the configuration in the `~/.source_cloud_exported_variables` file.
```
Comment (klgill) [inline docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc:105]: Hackfest comment for reference: https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABQ54dAl0

**Comment by klgill** [comment]
@jistr This is a PR that implements most of the Hackfest feedback. I added links to the comments in the Google doc for reference.

File: docs_user/modules/proc_adopting-the-networking-service.adoc [hunk 7]
```diff
@@ -70,7 +70,6 @@ $ openstack endpoint list | grep network
 
 | 6a805bd6c9f54658ad2f24e5a0ae0ab6 | regionOne | neutron      | network      | True    | public    | http://neutron-public-openstack.apps-crc.testing  |
 | b943243e596847a9a317c8ce1800fa98 | regionOne | neutron      | network      | True    | internal  | http://neutron-internal.openstack.svc:9696        |
-| f97f2b8f7559476bb7a5eafe3d33cee7 | regionOne | neutron      | network      | True    | admin     | http://192.168.122.99:9696                        |
 ----
```
Comment (klgill) [inline docs_user/modules/proc_adopting-the-networking-service.adoc:73]: Hackfest comment for reference: https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABQ54c_rI

File: docs_user/modules/proc_configuring-networking-for-control-plane-services.adoc [hunk 8]
```diff
@@ -15,6 +15,9 @@ For example,
 ```
 apiVersion: k8s.cni.cncf.io/v1
 kind: NetworkAttachmentDefinition
+metadata:
+  name: internalapi
+  namespace: openstack
 spec:
```
Comment (klgill) [inline docs_user/modules/proc_configuring-networking-for-control-plane-services.adoc:21]: @archana203 This doc update is from a bug you submitted: https://issues.redhat.com/browse/OSPRH-8468

File: docs_user/modules/proc_configuring-openshift-worker-nodes.adoc [hunk 9]
```diff
@@ -18,6 +18,7 @@ items:
 - apiVersion: nmstate.io/v1
   kind: NodeNetworkConfigurationPolicy
   spec:
+    desiredState:
       interfaces:
```
Comment (klgill) [inline docs_user/modules/proc_configuring-openshift-worker-nodes.adoc:22]: @archana203 This doc update is from a bug you submitted: https://issues.redhat.com/browse/OSPRH-8467

Comment (archana203) [reply docs_user/modules/proc_configuring-openshift-worker-nodes.adoc:22]: Lgtm

File: docs_user/modules/proc_configuring-networking-for-control-plane-services.adoc (hunk 8 above)
Comment (archana203) [reply docs_user/modules/proc_configuring-networking-for-control-plane-services.adoc:21]: Lgtm

**Review by archana203 — COMMENTED** [review]
Plz add similar lines here as well:
metadata:
  name: internalapi
  namespace: openstack

**Review by jistr — CHANGES_REQUESTED** [review]
Someone (i) will need to circle back to make these edits in tests.

There are two spots that i think need editing:

* I don't think we should be doing the `openstack-galera-network-isolation` and if yes it will need to be done on more places and probably by an engineer to cover the test suite too.
* The variables file and SRIOV variable snippets need to swap order to work as intended.

Other than those, LGTM :+1:

File: docs_user/modules/proc_adopting-image-service-with-nfs-backend.adoc (hunk 1 above)
Comment (klgill) [reply docs_user/modules/proc_adopting-image-service-with-nfs-backend.adoc:122]: I'll change it back to `openstack`. Do you want to follow up with Partheeban?

**Pull request merged**